Lambdas/GHHooks/Models/+PR.Label.swift

```diff
@@ -0,0 +1,24 @@
+
+extension PullRequest {
+    var knownLabels: [KnownLabel] {
+        self.labels.compactMap {
+            KnownLabel(rawValue: $0.name)
+        }
+    }
+}
+
+enum KnownLabel: String {
+    case semVerMajor = "semver-major"
+    case semVerMinor = "semver-minor"
+    case semVerPatch = "semver-patch"
+    case release = "release"
```

Comment:
Suggest adding an additional case for the `no-release-needed` label (which admittedly doesn't exist on all repos yet but will when I eventually get around to it) - and with it a check to ensure that it isn't present in combination with one of the release labels (not to mention that there aren't multiple release labels on a single PR in general).

Reply:
Hmm it will already do no releases for any labels other than these so I don't think adding that label will do much 🤔

Reply:
It'll do something if you use it to do the conflict check 🙂. Which actually is something worth having, trust me 😰. Ideally, we'd eventually have it so that merging a PR that has neither semver _nor_ no-release labels would result in the person who did the merge getting a warning notification (which we can't do right now since not all repos have the label).

Reply:
not sure if i you-noticed/i-mentioned but added that label too.